plugins/HydrogenImport/HydrogenImport.cpp

```diff
@@ -122,7 +122,7 @@ class NoteKey
 		else if ( sKey == "B" ) {
 			m_key = NoteKey::B;
 		} 
-        return m_key + (nOctave*12)+57;
+        return m_key + ((nOctave - FirstOctave) * 12) + 69;
```

Comment:
OK, so I made a simple project in Hydrogen, and it definitely seems that both the old and new versions are wrong, confirming what you said about C being mapped to A.

The Hydrogen "piano roll" has only labels for notes, but not octave numbers. The lowest note is C, and it only covers 7 octaves, so it could either start from C2, C1, C0 or C-1. I managed to somehow get the MIDI recording working and based on that it seems it goes from C-1 to B5, so I will change the code to match this range.

Reply:
The change was added (some time ago), along with a comment explaining what range of notes Hydrogen seems to be using, so I'm marking this as resolved.